changes/tasks.py

```diff
@@ -534,7 +534,43 @@ def switch_channel(self, change):
         }
         messagesets_rev = {v: k for k, v in messagesets.items()}
         params = {"identity": change.registrant_id, "active": True}
-        for sub in sbm_client.get_subscriptions(params)["results"]:
+        subscriptions = list(sbm_client.get_subscriptions(params)["results"])
+
+        # Check if they're in 1-2 messaging, in which case they cannot switch
+        short_names = [messagesets[sub["messageset"]] for sub in subscriptions]
+        if "whatsapp_momconnect_postbirth.hw_full.3" in short_names:
+            translation_lang = subscriptions[0]["lang"].lower().replace("_", "-")
+            with translation.override(translation_lang):
+                text = translation.ugettext(
+                    "We notice that you have been receiving MomConnect msgs on "
+                    "WhatsApp for children between 1 - 2. Messages for children "
+                    "between 1 - 2 are only available on WhatsApp - switching to "
+                    "SMS means you will not receive any messages. You can stop "
+                    "your MomConnect messages completely by replying 'STOP'"
+                )
+            reason = change.data and change.data.get("reason")
+            if reason != "whatsapp_unsent_event":
+                utils.ms_client.create_outbound(
+                    {
+                        "to_identity": change.registrant_id,
+                        "content": text,
+                        "channel": "WHATSAPP",
+                        "metadata": {
+                            "template": {
+                                "name": "important_info",
```

Comment:
does this template name need to change?

Reply:
Ah, yes it does, but I think that should happen on the other PR (since that's where this change is from)